tests/integ_tests/workflow_tests.rs

```diff
@@ -85,56 +83,6 @@ async fn parallel_workflows_same_queue() {
     core.run_until_done().await.unwrap();
 }
 
-static RUN_CT: AtomicUsize = AtomicUsize::new(0);
-
-pub(crate) async fn cache_evictions_wf(command_sink: WfContext) -> WorkflowResult<()> {
-    RUN_CT.fetch_add(1, Ordering::SeqCst);
-    command_sink.timer(Duration::from_secs(1)).await;
-    Ok(().into())
-}
-
-#[tokio::test]
-async fn workflow_lru_cache_evictions() {
```

Comment:
This test is removed now since it depended on having only 1 poller to work. It's covered by unit tests anyway.